integration-test/tests/bitwarden.macos.spec.js

```diff
@@ -0,0 +1,245 @@
+import {constants} from '../helpers/mocks.js'
+import {createAutofillScript, forwardConsoleMessages, mockedCalls} from '../helpers/harness.js'
+import {createWebkitMocks, macosContentScopeReplacements} from '../helpers/mocks.webkit.js'
+import {createAvailableInputTypes} from '../helpers/utils.js'
+import {loginPage, overlayPage} from '../helpers/pages.js'
+import {expect, test as base} from '@playwright/test'
+
+/**
+ *  Tests for various Bitwarden scenarios on macos
+ */
```

Comment:
I've moved all the bitwarden stuff to a separate file to speed up the test suite. Here we have slow tests (because of timeouts) so moving it to a separate file speeds everything up (other tests can run in parallel).